### PR TITLE
Add support to use llvm-project repo for building swift

### PR DESCRIPTION
**Pull request by shahmishal**

### CODE REVIEW

**Comment by shahmishal** [comment]
@swift-ci test

**Comment by benlangmuir** [comment]
> FAIL: Sourcery, 5.1

This is an existing issue we tried to fix here https://github.com/apple/swift-source-compat-suite/pull/390

File: common.py [hunk 1]
```diff
@@ -58,8 +54,7 @@
         'swift-corelibs-xctest': 'swift-4.2-branch',
     },
     'swift-5.0-branch': {
-        'llvm': 'swift-5.0-branch',
-        'clang': 'swift-5.0-branch',
+        'llvm-project': 'swift/swift-5.0-branch',
         'compiler-rt': 'swift-5.0-branch',
```
Comment (xedin) [inline common.py]: @shahmishal Looks like other configs also remove `compiler-rt`, not sure if that's important or not.

Comment (shahmishal) [reply common.py]: I missed `compiler-rt` in this config

**Pull request merged**